src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -1225,15 +1281,36 @@ public void tryMarkRemoved(Topology topology, Id target)
         if (node.commandStores().count() == 0) return; // when starting up stores can be empty, so ignore
         Ranges ranges = topology.rangesForNode(target);
         if (ranges.isEmpty()) return;
-        tryMarkRemoved(ranges, 0).begin(node().agent());
+        long startNanos = Clock.Global.nanoTime();
+        exclusiveSyncPointWithRetries(ranges, 0)
+        .flatMap(sp -> shardDurabilityWithRetries(sp, 0))
```

Comment:
> 7) when nodes leave the cluster we did not start durability sync (this isn’t working, but thats a different issue… durability sync requires ALL which isn’t possible)

This is failing 100% of the time, but thats a bug in durability scheduling... it requires `ALL`, which isn't possible as we *removed* a node... it also means downed nodes block this logic as well... spoke with Benedict and deferring fixing the durability issue for now, just enabling the attempts so when its fixed this logic works fine